.github/workflows/tests.yml

```diff
@@ -0,0 +1,29 @@
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+        with:
+          repository: alphagov/govuk-content-schemas
+          ref: deployed-to-production
+          path: govuk-content-schemas
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: .ruby-version
```

Comment:
As this is the default we could drop this line?

Reply:
NIIIICE! I failed at reading the docs. 😭